.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: python
+
+python:
+  - "2.6"
```

Comment:
The setup.py file says it support 2.6 under classifiers unless that isn't what the section is for?